docs/semgrep-code/supported-languages-python.md

```diff
@@ -0,0 +1,195 @@
+---
+slug: supported-languages-python
+id: supported-languages-python
+append_help_link: true
+description: >-
+  Detailed documentation for Semgrep's Python support. 
+hide_title: true
+tags:
+    - Semgrep Code 
+title: Supported languages for Python
+---
+# Python
+
+:::tip 
+Semgrep’s Python coverage leverages framework-specific analysis capabilities that are not present in OSS. As a result, many framework specific Pro rules will **fail** to return findings if run on OSS. To ensure full security coverage, run: `semgrep login && semgrep ci`
+:::
+
+## Analysis Capabilities for Python
+* Inter-file analysis (cross-file)
+* Inter-procedural analysis (cross-function)
+* Framework-specific control flow analysis 
+
+## Coverage 
+_Our coverage ethos: comprehensive and accurate detection of common, OWASP Top 10 issues in source code_
+
+
+Semgrep supports **over 100** common Python frameworks and libraries, including the 3 most popular frameworks: 
+
+<table>
+    <thead><tr>
+        <td><strong>Framework / library</strong></td>
+        <td><strong>Category</strong></td>
+    </tr></thead>
+    <tbody>
+    <tr>
+        <td>Framework / library</td>
+        <td>Django</td>
+        <td>Flask</td>
+        <td>FastAPI</td>
+    </tr>
+    <tr>
+        <td>Category</td>
```

Comment:
```suggestion
        <td colspan="3">Category</td>
```
May resolve the following blank cells
![image](https://github.com/user-attachments/assets/54232ded-3bc0-43ba-82e1-0054e40f6b86)